src/test/java/edu/hawaii/its/api/controller/TestGroupingsRestControllerv2_1.java

```diff
@@ -822,14 +829,14 @@ public void enableDisablePreferencesAndSyncDestsPassTest() throws Exception {
     @Test
     public void enableDisablePreferencesFailTest() throws Exception {
         try {
-            mapGSRs(API_BASE + "groupings/" + GROUPING + "/preferences/" + OPT_IN + "/disable", "put", uhUser02);
+            mapGSRs(API_BASE + "groupings/" + GROUPING + "/preferences/" + OPT_IN + "/disable", "put", bogusUser);
             fail("Shouldn't be here.");
```

Comment:
Just a suggestion, we can write why the test should not reach this fail.

Reply:
Wow-what a careless mistake.